tests/integration/components/donation-goal-edit-test.js

```diff
@@ -14,36 +18,32 @@ function setHandlers(context, { saveHandler = function() {}, cancelHandler = fun
   context.set('cancelHandler', cancelHandler);
 }
 
-moduleForComponent('edit-donation-goal', 'Integration | Component | donation goal edit', {
+moduleForComponent('donation-goal-edit', 'Integration | Component | donation goal edit', {
   integration: true,
   beforeEach() {
     this.set('donationGoal', mockGoal);
     setHandlers(this);
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext();
   }
 });
 
-test('it renders', function(assert) {
-  assert.expect(1);
-
-  this.render(hbs`{{donation-goal-edit donationGoal=donationGoal cancel=cancelHandler save=saveHandler}}`);
-
-  assert.equal(this.$('.donation-goal-edit').length, 1);
-});
-
 test('it renders cancel button, when canCancel is true', function(assert) {
   assert.expect(1);
 
-  this.render(hbs`{{donation-goal-edit donationGoal=donationGoal canCancel=true cancel=cancelHandler save=saveHandler}}`);
+  page.render(hbs`{{donation-goal-edit donationGoal=donationGoal canCancel=true cancel=cancelHandler save=saveHandler}}`);
```

Comment:
Haven't seen this use before (`page.render`).  Guessing you found this in the documentation?